packages/rust.rb

```diff
@@ -71,4 +66,20 @@ def self.install
     IO.write("#{CREW_DEST_PREFIX}/etc/bash.d/rust", @rustcompletionenv)
     system "#{CREW_DEST_PREFIX}/share/cargo/bin/rustup completions bash > #{CREW_DEST_PREFIX}/share/bash-completion/completions/rustup"
   end
+  
+  def self.remove
+    config_dirs = %W[#{HOME}/.rustup #{CREW_PREFIX}/share/rustup #{HOME}/.cargo #{CREW_PREFIX}/share/cargo]
+    config_dirs.each do |config_dir|
+      next unless Dir.exist? config_dir
+
+      print "\nWould you like to remove #{config_dir}? [y/N] "
+      case $stdin.getc
+      when 'y', 'Y'
+        FileUtils.rm_rf config_dir
+        puts "#{config_dir} removed.".lightred
+      else
+        puts "#{config_dir} saved.".lightgreen
+      end
+    end
+  end
```

Comment:
This doesn't work like you might expect.  The first prompt works but the second prompt is bypassed.  Check for yourself to see what I mean.  Take a look at github_desktop.rb for an example of what works.

Reply:
Also, a single prompt would be less verbose than four separate prompts. :)

Reply:
I can't actually test this as this is dependent upon NOSYMFOLLOW keeping those dirs from being deleted. Does this work for you?

Reply:
Nice!